net-utils/src/lib.rs

```diff
@@ -484,8 +527,12 @@ pub fn multi_bind_in_range(
             port
         }; // drop the probe, port should be available... briefly.
 
+        let config = SocketConfig {
+            reuseaddr: true,
+            reuseport: true,
```

Comment:
This should be reuseport: false

Reply:
I think this comment is no longer relevant, can you check again?

Reply:
I mean, I think this should still be false, but there are other places too in net-utils that set reuseport=true for no reason. We don't have to fix it all in this PR, it's pre existing bugs, they can be fixed in followup PRs.